debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/DefaultWriteModelStrategy.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import org.bson.BsonDocument;
+
+import com.mongodb.client.model.WriteModel;
+
+import io.debezium.connector.mongodb.converters.sink.SinkDocument;
+
+public class DefaultWriteModelStrategy implements WriteModelStrategy {
```

Comment:
Do we need a separate class?

Reply:
Yes, there are other scenarios to be implemented: Insert/Append-only, Update (the doc content / changed fields instead of replacing the whole document), Delete, etc

Filed https://issues.redhat.com/browse/DBZ-7997 for that.